### PR TITLE
Update sending emails from a service tutorial

### PR DESCRIPTION
**Pull request by mindula**
$title.

### CODE REVIEW

File: swan-lake/integration-tutorials/sending-emails-from-a-service.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-title: Send emails from a service
-permalink: /learn/send-emails-from-a-service/
+layout: ballerina-sending-emails-from-a-service-left-nav-pages-swanlake
+title: Sending emails from a service
+permalink: /learn/sending-emails-from-a-service/
 description: Integration tutorial for sending emails from a service.
 keywords: ballerina, programming language, client, restful-api, integration, smtp, email
-active: send-emails-from-a-service
+active: sending-emails-from-a-service
 intro: This tutorial helps you understand the basics of Ballerina constructs, which allow you to do client calls and develop RESTful APIs.
```
Comment (MaryamZi) [inline swan-lake/integration-tutorials/sending-emails-from-a-service.md:8]: Shall we update this to reflect sending emails?

File: swan-lake/integration-tutorials/sending-emails-from-a-service.md [hunk 2]
```diff
@@ -630,7 +630,7 @@ Let's test the use case by sending a request to the service.
 
 Download the JAR file for the [backend service](https://github.com/ballerina-guides/integration-tutorials/blob/main/backends/hospital-service/hospitalservice.jar) and execute the following command to start the service:
 
-```
+```bash
```
Comment (MaryamZi) [inline swan-lake/integration-tutorials/sending-emails-from-a-service.md:633]: These also have to be indented?

File: swan-lake/integration-tutorials/sending-emails-from-a-service.md [hunk 3]
```diff
@@ -614,10 +614,10 @@ function getEmailContent(int appointmentNumber, Appointment appointment, Payment
 
 You can run this service by navigating to the project root and using the `bal run` command.
 
-```
-send-emails-from-a-service$ bal run
+```bash
```
Comment (mindula) [inline swan-lake/integration-tutorials/sending-emails-from-a-service.md:617]: @MaryamZi Should this be indented too?

Comment (MaryamZi) [reply swan-lake/integration-tutorials/sending-emails-from-a-service.md:617]: Yeah, Anupama's suggestion was indent code blocks that were under bullet/numbered points.

Comment (MaryamZi) [reply swan-lake/integration-tutorials/sending-emails-from-a-service.md]: My bad, this is a heading, right? Sorry, we don't need indentation here.

**Review by MaryamZi — APPROVED** [review]

**Pull request merged**